src/components/date-input/README.njk

```diff
@@ -55,7 +55,7 @@
         text: 'No'
       },
       {
-        text: 'Optional name. This is used to compose the name attribute for each item.'
+        text: 'Optional prefix. This is used to prefix each `item.name`.'
```

Comment:
I wonder if it's worth mentioning that they'll be joined using "-"?

Reply:
Done 👍  I thought initially about adding it but it felt like too much about the implementation. But it's probably useful information.